src/main/java/com/GotYourBac/gotYourBac/controllers/AppErrorController.java

```diff
@@ -1,17 +1,31 @@
 package com.GotYourBac.gotYourBac.controllers;
 
 import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.http.HttpServletRequest;
 
-//https://www.baeldung.com/spring-boot-custom-error-page
 @Controller
 public class AppErrorController implements ErrorController {
 
-    //TODO: add some logging later
     @GetMapping("/error")
-    public String handleError() {
-        //do something like logging
+    public String handleError(HttpServletRequest request) {
+        Object status = request.getAttribute(RequestDispatcher.ERROR_STATUS_CODE);
```

Comment:
Pass request through a Model M into error than parse that information to have one dynamic error.html